pages/common/dolt.md

```diff
@@ -1,13 +1,33 @@
 # dolt
 
-> An SQL database that you can fork, clone, branch, merge, push and pull just like a Git repository.
+> A SQL database that you can fork, clone, branch, merge, push, and pull just like a Git repository.
```

Comment:
"An" in correct. The abbreviation is pronounced "es-qyu-el"

Reply:
SQL has two common pronunciations, and the correct article (“a” vs. “an”) depends on which one we use:
1. Pronounced “sequel”
   - Common in phrases like “SQL database” -> pronounced “sequel database”.
   - Because sequel starts with a consonant sound, we use a:
     - a SQL database

2. Pronounced “ess-cue-el”
   - Common when referring to queries or the language explicitly.
   - Because ess starts with a vowel sound, we use an:
     - an SQL query

Reply:
Oh right I forgot about that pronounciation fight. I would personally go with the pronounciation that is directly apparent for anyone not aware of the historical context: "ess-cue-el".
But I'll leave that up in the air.